src/main/java/net/william278/velocitab/packet/ScoreboardManager.java

```diff
@@ -108,12 +109,20 @@ public void registerPacket() {
                     .mapping(0x55, MINECRAFT_1_17, false)
                     .mapping(0x58, MINECRAFT_1_19_1, false)
                     .mapping(0x56, MINECRAFT_1_19_3, false)
-                    .mapping(0x5A, MINECRAFT_1_19_4, false)
-                    .register();
+                    .mapping(0x5A, MINECRAFT_1_19_4, false);
+            packetRegistration.register();
         } catch (Throwable e) {
             plugin.log(Level.ERROR, "Failed to register UpdateTeamsPacket", e);
         }
     }
 
+    public void unregisterPacket() {
+        try {
+            packetRegistration.unregister();
```

Comment:
can we null-check this in the event the plugin terminates before registration? just a little guard clause would probably be fine.